openqa-mail-notification/mail_notification.rb

```diff
@@ -78,11 +78,17 @@ def send_notification(to, subject, message)
 
 Hash[VERSIONS.split.each_slice(2).to_a].each_pair do |version, group|
   build = get_build_information(version, group)
+
+  unless build['state'] == 'done'
+    @logger.info("Build not done yet for #{version}...")
+    next
+  end
+
   last_build = DateTime.parse(build['t_finished'])
   frequency_minutes_ago = DateTime.now - (FREQUENCY / 1440.0)
   result = last_build >= frequency_minutes_ago
 
-  unless result && build['state'] == 'done'
+  unless build['state'] == 'done'
```

Comment:
If the variable `result` is no longer used, we could also remove the lines where `last_build`, `frequency_minutes_ago` and `result` get assigned.

Reply:
that was a mistake. pushed. good catch!